src/App.js

```diff
@@ -1,13 +1,66 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
 
 import todos from './api/todos';
```

Comment:
```suggestion
import fetchTodos from './api/fetchTodos';
```
Название todos не дает понимания что делает эта функция